R/resample.R

```diff
@@ -131,6 +131,11 @@ resample = function(learner, task, resampling, measures, weights = NULL, models
     printResampleFormatLine("Measures:", measure.lognames)
   }
 
+  # FIXME
+  # iter.results contains only thresholds of 0.5
+
+  # The learner going in here should contain a $predict.threshold slot if tune.threshold = TRUE during tuning
```

Comment:
Should this be a FIXME or WONTFIX? Not a fan of adding FIXMEs in PRs.

Reply:
This behaviour should also be documented.

Reply:
oh yeah, this was just some internal leftover of myself. Cleared. Thanks!